ec/resources/rulesengine/v4policy.go

```diff
@@ -361,6 +370,9 @@ func addPolicy(
 
 func cleanPolicyForTerrafomState(val interface{}) string {
 	policy := val.(string)
+	if len(policy) == 0 {
```

Comment:
What would happen if input has an empty json body like {}?

Reply:
@changgyu-oh This caused a panic... fixed, see updated code

Reply:
@changgyu-oh @wdawg33 a bigger question is... technically we are capable of checking:

1. if the policy is empty
2. if the policy has no rules
3. if the policy has one or more rules with no matches
4. If the policy has one or more matches with no features

Should I add in this validation? I think it can be pretty straightforward, or we can let the API handle anything beyond empty policies.

Reply:
I have decided to leave just the empty policy and empty json validation